addon/components/flash-message.js

```diff
@@ -24,6 +24,7 @@ export default Component.extend({
   layout,
   active: false,
   messageStyle: 'bootstrap',
+  classNames: 'flash-message',
```

Comment:
I believe that `classNames` still needs to be an array even if we are only adding 1 class.
`classNames: ['flash-message'],`